src/Summoner/Source.hs

```diff
@@ -18,26 +18,34 @@ data Source
     = Url Text
     -- | File path to the local source file.
     | File FilePath
+    -- | Link to source source file.
```

Comment:
Double `source` here. Probably a typo :slightly_smiling_face: Maybe better to say `Link to external file` or something like that? :thinking:

Reply:
oops! 🤦‍♂️ agreed on term sounds much better.